src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.main.allow-circular-references=true
```

Comment:
why is this needed?

Reply:
if we have a circular reference we should fix the code, probably we can improve the design and decouple services, I don't like to have this setting.